### PR TITLE
improve readme

### PR DESCRIPTION
**Pull request by z44d**
Add table of contents to access what we want faster and more.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -27,7 +27,45 @@ Fast and lightweight key-value database library.
...
+  - [Multi Get](#multi-get)
+  - [Multi Set](#multi-set)
+  - [Multi Delete](#multi-delete)
+  - [Cleanup](#cleanup)
+  - [Vacuum](#vacuum)
+  - [Flush](#flush)
+  - [Execute](#execute)
+  - [Backup](#backup)
+  - [Restore](#restore)
+  - [Close](#close)
```
Comment (vwh) [inline README.md]: Thanks for your clarification
It looks better and easy to navigate 

But the table of contents is big 
I don't think we should list each method we can stick to point to the API reference it self

Comment (z44d) [reply README.md]: Done

**Comment by z44d** [comment]
> 🎉 This issue has been resolved in version 1.16.0 🎉
> 
> The release is available on:
> 
> * [npm package (@latest dist-tag)](https://www.npmjs.com/package/miftahdb/v/1.16.0)
> * [GitHub release](https://github.com/miftahDB/miftahdb/releases/tag/v1.16.0)
> 
> Your **[semantic-release](https://github.com/semantic-release/semantic-release)** bot 📦🚀

huh? it's not a issue

**Comment by vwh** [comment]
> > 🎉 This issue has been resolved in version 1.16.0 🎉
> > The release is available on:
> > 
> > * [npm package (@latest dist-tag)](https://www.npmjs.com/package/miftahdb/v/1.16.0)
> > * [GitHub release](https://github.com/miftahDB/miftahdb/releases/tag/v1.16.0)
> > 
> > Your **[semantic-release](https://github.com/semantic-release/semantic-release)** bot 📦🚀
> 
> huh? it's not a issue

nvm
github-actions bot pushed a new release

**Pull request merged**